src/campaign_server/campaign_server.cpp

```diff
@@ -881,6 +881,11 @@ void server::handle_upload(const server::request& req)
 			(*campaign).add_child("feedback", url_params);
 		}
 
+		(*campaign).clear_children("translation");
+		for(const config& locale_params : upload.child_range("translation")) {
+			(*campaign).add_child("translation", locale_params);
+		}
+
```

Comment:
This is the best place to perform the sanitation pass I refer to in my comment on `find_translations()`.

Reply:
Ok